tests/nginx.nix

```diff
@@ -186,9 +186,6 @@ in {
       # Prepare change to mainline nginx. We are not interested in testing mainline itself here.
       # We only need it as a different version so we can test binary reloading.
       server.execute("ln -sfT ${pkgs.nginxMainline} /etc/nginx/running-package")
-      # Mainline doesn't know about our custom remote_addr_anon variable, patch our config.
```

Comment:
Can we please also add explicit tests that our default config is in fact anonymizing the IPs?

Reply:
Added